src/EditorFeatures/TestUtilities/Roslyn.Services.Test.Utilities.csproj

```diff
@@ -53,7 +53,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
     <PackageReference Include="Nerdbank.FullDuplexStream" Version="$(NerdbankFullDuplexStreamVersion)" />
     <PackageReference Include="BasicUndo" Version="$(BasicUndoVersion)" />
-    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="$(SQLitePCLRawbundle_greenVersion)"/>
+    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="$(SQLitePCLRawbundle_greenVersion)" />
```

Comment:
lol.  Did this annoy you or something? :D

Reply:
I don't really do anything..Maybe it annoys my Visual Studio it got changed 😆